aries-site/src/pages/components/pagination.mdx

```diff
@@ -1,42 +1,69 @@
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/togglegroup/utils.js',
+  ]}
+>
+  <ToggleGroupViews />
+</Example>
 
-### Number of results per page
+## Anatomy
 
-A reasonable starting point for page size is 20 records. This is the default "step" (page size) on Grommet DataTable, List, and Pagination. This size strikes a balance between providing a user enough information to consume or find records of interest without being too much information leading to cognitive overload.That said, the default number of records per page should be tailored to the data set and determined by the UX designer. For example, a data collection displaying log files or notifications may choose a large page size as it allows a user to scan and understand activity in a efficient manner. Whereas as data collection presenting items in a news feed may only need to present a handful of records at a time.
+<Example
+  caption="Diagram illustrating the parts composing Pagination."
+  plain
+>
+  <PaginationAnatomy />
+</Example>
 
-If there is only one page of items, it is optional to display the pagination component and summary of results.
+| Label | Region          | Purpose                                                                            | Required | Notes                                                                                                                                      |
+| :---: | --------------- | ---------------------------------------------------------------------------------- | :------: | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Container**   | Organizes its children, where they appear, how they adapt responsively.            | Required | --                                                                                                                                         |
+| **2** | **Summary**     | Provide clarity on what range of items are in view.                                | Optional | --                                                                                                   |
+| **3** | **StepOptions** | Displays the current number of items within a page and allows the user to modify.  | Optional | Step defaults to 10. Dropdown options default to 10, 25, 50, 100. The default step and step options can be customized to fit the use case. |
+| **4** | **Navigation**  | Allow user to jump to a specific page.                                             | Required | --                                                                               |
 
-### Placement of pagination component
+## Types
 
-The pagination component should be placed underneath and justified to the end of the content it is paginating.
+### Pagination with Standard Navigation
```

Comment:
Maybe just call this `Default`? Feels like we typically use that terminology instead of "standard"

Feels like the other words are redundant since we're on "Pagination" page

Reply:
I am ok with calling it default, however I would keep the navigation word as well so we don't mistake the full pagination component (summary, StepOptions and Navigation) with the navigation itself. So I would call it "Default Navigation"